### ISSUE TITLE
ember-cli install fails

### ISSUE THREAD

**Issue by knownasilya** [lytics/ember-data.model-fragments#45]
Here's the error after `ember install:addon ember-data-model-fragments`:

```
version: 0.1.5
Installing packages for tooling via npm.sh: ./node_modules/bower/bin/bower: No such file or directory
ember-data-model-fragments@0.2.4 postinstall: `./node_modules/bower/bin/bower install`
Exit status 127
Error: ember-data-model-fragments@0.2.4 postinstall: `./node_modules/bower/bin/bower install`
Exit status 127
    at EventEmitter.<anonymous> (/Users/iradchenko/workspace/mapgeo2/dashboard/node_modules/ember-cli/node_modules/npm/lib/utils/lifecycle.js:212:16)
    at EventEmitter.emit (events.js:98:17)
    at ChildProcess.<anonymous> (/Users/iradchenko/workspace/mapgeo2/dashboard/node_modules/ember-cli/node_modules/npm/lib/utils/spawn.js:14:12)
    at ChildProcess.EventEmitter.emit (events.js:98:17)
    at maybeClose (child_process.js:743:16)
    at Process.ChildProcess._handle.onexit (child_process.js:810:5)
```

**Comment by slindberg** [comment]
Love that you can't truly test these things until post-publish. Weeee!

**Comment by knownasilya** [comment]
Yeah, good thing we have the patch number in semver :+1:

**Comment by knownasilya** [comment]
Maybe use this feature: https://github.com/ember-cli/ember-cli/pull/2805
Plus something like this https://gist.github.com/stravid/a3a56d8eadc6251c0be9#file-blueprint-js maybe

**Comment by slindberg** [comment]
That was the intention. The `postinstall` hook in `package.json` that is screwing this up has been around since the beginning, I do it with all projects.

**Comment by slindberg** [comment]
This should be fixed in v0.2.5.
